ui/src/components/yaml/styles.module.less

```diff
@@ -223,7 +221,9 @@
       justify-content: space-between;
       align-items: center;
       padding: 12px;
```

Comment:
The border-bottom color may not match the gradient's end color, which could result in an inconsistent visual appearance.